Assets/Scripts/Actor/EnemyLiving.cs

```diff
@@ -0,0 +1,43 @@
+﻿using BulletManagement;
+using Player;
+using UnityEngine;
+
+namespace Actor
+{
+    public class EnemyLiving : Living
+    {
+        public int hitScoreBonus = 1;
+        
+        public int deathScoreBonus = 1000;
+
+        private new void Start()
+        {
+            // Set the layer from which to take damage to the player's bullets.
+            // We do this in Start() instead of hiding the parent field with the "new" keyword
+            // because Unity complains about serialization of multiple of the same field if we do so.
+            damageLayer = 11;
+            base.Start();
+        }
+
+        protected override void Die()
+        {
+            PlayerStats.Score += deathScoreBonus;
+            base.Die();
+        }
+        
+        private void OnTriggerEnter2D(Collider2D other)
+        {
+            GameObject go = other.gameObject;
+            if (go.layer == damageLayer)
+            {
+                TakeDamage(10);
+                PlayerStats.Score += hitScoreBonus;
```

Comment:
The enemy has a cool down equivalent to the color effect (which is set to 0.5f.) Do we want the score to also accommodate the cooldown?

Reply:
I asked @hirudan about this, and he thought it would be better if we just grant the score bonus per bullet hit. This will potentially need to be changed if we end up with different ships that fire different bullet densities/frequencies, but that's probably a wave 2 thing.